modular_zubbers/code/modules/customization/species/lycans/_cursekin_species.dm

```diff
@@ -0,0 +1,84 @@
+/datum/species/human/cursekin
+	name = "\improper Cursekin"
+	id = SPECIES_CURSEKIN
+	inherent_traits = list(
+		TRAIT_LYCAN,
+		TRAIT_MUTANT_COLORS, // More customization options.
+	)
+	mutant_bodyparts = list()
```

Comment:
why have this  be empty? if it's not doing anything remove it, unless the base type has some?